src/code/z_room.c

```diff
@@ -114,7 +114,7 @@ s32 Room_HandleLoadCallbacks(PlayState* play, RoomContext* roomCtx) {
 
             if (((play->sceneNum != SCENE_IKANA) || (roomCtx->currRoom.num != 1)) &&
                 (play->sceneNum != SCENE_IKNINSIDE)) {
-                play->envCtx.lightSettingOverride = 0xff;
+                play->envCtx.lightSettingOverride = 0xFF;
```

Comment:
This ought to get a define, OoT has `LIGHT_SETTING_OVERRIDE_NONE` in `z64environment.h`.

Reply:
I want to keep this PR focused to just hex numbers, so we'll add it to the to-do list